MetaMask/metamask-extension
# Add Coinbase Pay as fiat onramp option

## Explanation
Adding support for a new fiat on-ramp option using Coinbase Pay. 



## More Information



## Screenshots/Screencaps



### Before

<img width="300" alt="image" src="https://user-images.githubusercontent.com/30222060/167439857-801e6cb6-15b2-473a-bb4d-b3e0bb769c39.png">



### After

| Light | Dark |
|---|---|
| <img width="300" alt="image" src="https://user-images.githubusercontent.com/30222060/167434857-93059776-ca4c-44cd-8378-d6c7991b73b6.png"> | <img width="300" alt="image" src="https://user-images.githubusercontent.com/30222060/167434448-196dcee2-321f-4082-866a-f5cd071288d1.png"> |







## Manual Testing Steps

- Open the MetaMask extension
- Click on Buy on the main page _(must be Ethereum, Avalanche C Chain, or Celo networks)_
- Click on "Continue to Coinbase Pay"


_Should open a new tab. Continuing in the Coinbase Pay onramp flow:_
- Sign into a coinbase.com account 
- Select an asset 
- Input an amount
- Verify wallet address is populated on the preview screen



## Pre-Merge Checklist
